pass/add_creds_gpg.sh

```diff
@@ -73,14 +73,14 @@ cleanup() {
 trap cleanup EXIT
 
 gpg_sb() {
+  mkdir -p "${TMP_GPG}"
+  chmod 700 "${TMP_GPG}"
```

Comment:
It should not be required to do this more than once. Moving both commands to line 69 might be easier.

Reply:
done!